Makefile

```diff
@@ -0,0 +1,85 @@
+NAMESPACE := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["namespace"])')
+NAME := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["name"])')
+VERSION := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["version"])')
+MANIFEST := build/collections/ansible_collections/$(NAMESPACE)/$(NAME)/MANIFEST.json
+
+ROLES := $(wildcard roles/*)
+DOCS := $(wildcard docs/*)
+META := $(wildcard meta/*)
+PLAYBOOKS := $(wildcard playbooks/*)
+METADATA := galaxy.yml LICENSE README.md requirements.yml
+DEPENDENCIES := $(METADATA) $(foreach ROLE,$(ROLES),$(wildcard $(ROLE)/*/*)) $(DOCS) $(foreach DOC,$(DOCS),$(wildcard $(DOC)/*)) $(META) $(PLAYBOOKS) $(foreach PLAY,$(PLAYBOOKS),$(wildcard $(PLAY)/*)) $(foreach PLAY,$(PLAYBOOKS),$(wildcard $(PLAY)/*/*))
+
+PYTHON_VERSION = $(shell python -c 'import sys; print("{}.{}".format(sys.version_info.major, sys.version_info.minor))')
+COLLECTION_COMMAND ?= ansible-galaxy
+SANITY_OPTS =
+TEST =
+PYTEST = pytest -n 4 --boxed -v
+
+default: help
+help:
+	@echo "Please use \`make <target>' where <target> is one of:"
+	@echo "  help             to show this message"
+	@echo "  info             to show infos about the collection"
+	@echo "  lint             to run code linting"
+	@echo "  test             to run unit tests"
+	@echo "  livetest         to run test playbooks live (without vcr)"
+	@echo "  sanity           to run santy tests"
+	@echo "  setup            to set up test, lint"
+	@echo "  test-setup       to install test dependencies"
+	@echo "  test_<test>      to run a specific unittest"
+	@echo "  livetest_<test>  to run a specific unittest"
+	@echo "  record_<test>    to (re-)record the server answers for a specific test"
+	@echo "  clean_<test>     to run a specific test playbook with the teardown and cleanup tags"
+	@echo "  dist             to build the collection artifact"
+
+info:
+	@echo "Building collection $(NAMESPACE)-$(NAME)-$(VERSION)"
+	@echo "  roles:\n $(foreach ROLE,$(notdir $(ROLES)),   - $(ROLE)\n)"
+
+lint: $(MANIFEST)
+	yamllint -f parsable roles
+	ansible-playbook --syntax-check roles/*.yaml | grep -v '^$$'
+	black . --diff --check
+
+sanity: $(MANIFEST)
+	# Fake a fresh git repo for ansible-test
+	cd $(<D) ; git init ; echo tests > .gitignore ; ansible-test sanity $(SANITY_OPTS) --python $(PYTHON_VERSION)
```

Comment:
You can surely test those lintings, but otoh molecule does proper linting already, right?

Maybe keep the (collection) sanity, as molecule is a test for role afaik.